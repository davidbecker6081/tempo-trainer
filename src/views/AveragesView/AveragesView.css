.averages-view {
  display: flex;
  align-items: center;
  flex-direction: column;
  width: 25%;
}

.average-view h3 {
  margin-top: 0;
}
