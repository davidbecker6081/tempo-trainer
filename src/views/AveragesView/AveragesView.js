import React from 'react';
import Average from '../../components/Average/Average';
import './AveragesView.css';

const AveragesView = ({ dataHelper }) =>
  (
    <section>
      <Average dataHelper={dataHelper} channelSet="power" />
      <Average dataHelper={dataHelper} channelSet="heartRate" />
      <Average dataHelper={dataHelper} channelSet="cadence" />
      <Average dataHelper={dataHelper} channelSet="temperature" />
      <Average dataHelper={dataHelper} channelSet="elevation" />
      <Average dataHelper={dataHelper} channelSet="speed" />
    </section>
  );

export default AveragesView;
