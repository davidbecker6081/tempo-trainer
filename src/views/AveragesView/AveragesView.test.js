import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import AveragesView from './AveragesView';
import DataHelper from '../../components/DataCleaner/DataCleaner';
import workoutData from '../../__mock__/workout-data.json';

describe('AveragesView', () => {
  let wrapper;
  let mockProps;

  beforeEach(() => {
    mockProps = {
      dataHelper: {},
    };
  });

  describe('dataHelper is defined', () => {
    beforeEach(() => { mockProps.dataHelper = new DataHelper(workoutData); });
    it('should render correclty', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
