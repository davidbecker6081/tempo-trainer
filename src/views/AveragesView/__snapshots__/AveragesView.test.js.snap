// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AveragesView dataHelper is defined should render correclty 1`] = `undefined`;
